requirements.txt

```diff
@@ -0,0 +1,6 @@
+Django>=4.0,<5.0
+psycopg2-binary
+djangorestframework
+django-filter
+ipython
+Faker
```

Comment:
`pip freeze requirements.txt` would set specific versions of your environment automatically for your requirements file.